### DISCUSSION TITLE
Install does not work.

### DISCUSSION THREAD

**Discussion by andre-caldas** [BIMCoderLiang/LNLib#9] [General]
When running `make install`, what is installed is `gmock` and `gtest`. :-(
LNLib is not installed. :-(

I think there is no need to install `gmock` or `gtest`. But I do not understand anything about `cmake` in order to fix it.

I could try to migrate it to [meson](https://mesonbuild.com/) if you want.

**Comment by BIMCoderLiang** [comment]
#19
